nexus/auth/src/authz/api_resources.rs

```diff
@@ -885,6 +885,14 @@ authz_resource! {
     polar_snippet = FleetChild,
 }
 
+authz_resource! {
```

Comment:
I'm defining this authz resource, but am not convinced that I'm using it right (or at all). Do these only apply to resources that use the `lookup*` macros? Any guidance here would be appreciated.

Reply:
I'm looking at the target blueprint as an analog here because it's similarly global.

It looks like we *don't* use `authz_resource!` for this.  Instead, we have a synthetic resource called `BlueprintConfig`.  It has its own type here and a singleton instance on which we manually implement `PolarClass` and `AuthorizedResource`.  Then there's a snippet in omicron.polar about it.

That kind of seems like the way to go here.  `TargetRelease` isn't a type of "resource" in the same way that I think the `authz_resource!` macro means it (i.e., something you could CRUD, that has an id, for which instances might be missing, etc.)

Reply:
Thank you, replaced with a synthetic resource in 5fe4bb3.